tasks/abs.rb

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+require 'json'
+require 'net/http'
+require 'yaml'
+require 'solid_waffle'
+require 'pry'
+
+def platform_uses_ssh(platform)
+  uses_ssh = if platform !~ %r{win-}
+               true
+             else
+               false
+             end
+  uses_ssh
+end
+
+def token_from_fogfile
+  fog_file = File.join(Dir.home, '.fog')
+  raise "Cannot file fog file at #{fog_file}" unless File.file?(fog_file)
+  contents = YAML.load_file(fog_file)
+  token = contents.dig(:default, :vmpooler_token)
+  token
+end
+
+def provision(platform, inventory_location)
+  include SolidWaffle
+  uri = URI.parse("http://vcloud.delivery.puppetlabs.net/vm/#{platform}")
```

Comment:
@tphoney It doesn't look like this is provisioning vms from abs since you are not connecting to the abs service, but still to vmpooler.  I noticed you closed the ticket FM-7545, but it doesn't look like abs is supported as a provisioner yet or am I misunderstanding?